src/site/content/en/blog/coop-coep/index.md

```diff
@@ -272,18 +299,74 @@ The `endpoints` property specifies the URLs of one or more reporting endpoints.
 The endpoint must accept CORS if it's hosted on a different origin. The browser
 will send reports with a Content-Type of `application/reports+json`.
 
-An example payload looks like this:
+An example COEP report payload when cross-origin resource is blocked looks like
+this:
 
 ```json
 [{
-  age: 0,
-  body: {
-    'blocked-url': 'https://third-party-test.glitch.me/check.svg',
-    type: 'corp'
+  "age": 25101,
+  "body": {
+    "blocked-url": "https://third-party-test.glitch.me/check.svg?",
```

Comment:
Can we have a note saying that blocked-url will be removed soon, so it's recommended to use blockedURL?

Alternatively it may be good to omit blocked-url.